src/components/Link/index.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+/* This Link component provides adapters for usage with Material-UI */
```

Comment:
Can you comment that this came from https://github.com/mui-org/material-ui/blob/c795ca0822e2816a630a5654d5471fa04521d23c/examples/nextjs-with-typescript/src/Link.tsx? I think it's always helpful to have the context for where the code came from.

I'm still not entirely clear on what this provides. Is it just better styling when using Next links?

Reply:
@OzzieOrca Sure thing. I will comment some details for the Link component.
And you could refer to the routing page in [Material UI doc](https://next.material-ui.com/guides/routing/#next-js).

Reply:
@OzzieOrca It seems like the Material UI Link component has some issues with TypeScript, so I removed the component at this moment.